### ISSUE TITLE
"My Recipe" button looks like Disabled control but it is not.

### ISSUE THREAD

**Issue by SuryacharanM** [dotnet/maui#6598]
GitHubTags:
#A11yUsable; #.NET MAUI; #AILimited; #A11ySev4; #A11yTCS; #Android; #AndroidPhone;#GH_.NETMAUI_Android_Apr2022;

### Description

"My Recipe" button looks like Disabled control but it is not disabled.

### Steps to Reproduce

1. Install and open ".NET MAUI" app.
2. "Home Screen" will open.
3. Swipe till “My Recipe” and observe the issue.
![image](https://user-images.githubusercontent.com/86710788/165722278-d86250c7-6c67-4740-8046-e72a9a87b1ee.png)

### Version with bug

Unknown/Other (please specify)

### Last version that worked well

Unknown/Other

### Affected platforms

Android

### Affected platform versions

Android 11

### Relevant log output

```shell
"My Recipes" should look like a interactive control and should be bright.
```

**Comment by rachelkang** [comment]
Note to self: Spike various TabBar properties and explore different highlight colors for styles

**Comment by SuryacharanM** [comment]
Verified issue in ".NET MAUI" app. issue is fixed so closing the bug
Android Version: Android 11
Added attachment for references
![MicrosoftTeams-image (21)](https://user-images.githubusercontent.com/86710788/178657074-37cead37-a89f-426b-99e1-f22b98831f3b.png)
